docs/.vuepress/public/js/i18n.js

```diff
@@ -0,0 +1,82 @@
+var locale, dir, file_path, tip, outdated_tip, processing_tip, is_processing;
+
+function initTip(locale) {
+    switch (locale) {
+        case 'en':
+            outdated_tip = 'The translation of this page may not be up to date, please refer to the <a>English document</a> for the latest information. If there is a problem with the translation, please <a>let us know</a>.';
+            processing_tip = 'The translation of extend documents is still a work in progress, please refer to the <a>English document</a> for the latest information. If you have problems with the translation, please <a>let us know</a>.';
+            is_processing = false;
+            break;
+        case 'zh':
+            outdated_tip = '此页面的翻译可能更新不及时，最新信息请以<a>英文文档</a>为准。如果翻译有问题，请<a>告诉我们</a>。';
+            processing_tip = '当前文档仍在翻译中（进度 6/23），有关最新信息，请查阅<a>英文文档</a>。如果翻译有问题，请<a>告诉我们</a>。';
+            is_processing = true;
+            break;
+        case 'tr':
+            outdated_tip = 'The translation of this page may not be up to date, please refer to the <a>English document</a> for the latest information. If there is a problem with the translation, please <a>let us know</a>.';
```

Comment:
Would it be possible to use translated text for these?

Reply:
I don't speak Turkish, we can let @DursunCanPoyraz translate it later